rispy/parser.py

```diff
@@ -199,8 +199,6 @@ def _add_list_value(self, record: dict, name: str, value: Union[str, list[str]])
         except KeyError:
             record[name] = value_list
         except AttributeError:
-            if not isinstance(record[name], str):
```

Comment:
I deleted this statement; I couldn't think of a way to test it and didn't understand why it'd be relevant, but we can restore if needed. @J535D165

Reply:
Maybe this was a leftover from the past or a mistake from my side. It doesn't raise anything, so it's a bad practice. So I agree with removing.